src/app/app-routing.module.ts

```diff
@@ -1,15 +1,18 @@
 import { NgModule } from '@angular/core';
+import { AthletesComponent } from './athletes/athletes.component';
+
 import { RouterModule, Routes } from '@angular/router';
 import { LoginComponent } from "./login/login.component";
 import { AuthGuard } from "./helpers/auth.guard";
 import { HomeComponent } from "./home/home.component";
 import { SettingsComponent } from "./settings/settings.component";
 
 const routes: Routes = [
-  {path: '', component: HomeComponent, canActivate: [AuthGuard]},
+  {path: 'athletes' , component: AthletesComponent, canActivate:[AuthGuard]},
+  {path: 'home', component: HomeComponent, canActivate: [AuthGuard]},
   {path: 'login', component: LoginComponent},
   {path: 'settings', component: SettingsComponent, canActivate: [AuthGuard]},
-  {path: '**', redirectTo: ''}
+  {path: '**', redirectTo: 'home'}
```

Comment:
Redirect to '' y 'home' es lo mismo

Reply:
Es para tener más claro que home es donde se debe atterizar después del login. Igualmente si es necesario podemos borrar home y atterizar en athletes.

Reply:
vale, me parece bien dejarlo así